src/Scope.cc

```diff
@@ -117,7 +115,7 @@ const IDPtr& lookup_ID(const char* name, const char* curr_module,
 	bool need_export = check_export && (ID_module != GLOBAL_MODULE_NAME &&
 	                                    ID_module != curr_module);
 
-	for ( int i = scopes.length() - 1; i >= 0; --i )
+	for ( int i = scopes.size() - 1; i >= 0; --i )
```

Comment:
This could use a reverse iterator (`rbegin()`/`rend()`) instead of direct indexing.

Reply:
Roger, now done.